src/content/en/updates/2017/09/autoplay-policy-changes.md

```diff
@@ -221,7 +221,9 @@ will need to call <code>resume()</code> after a user gesture is received.
 
 If you create your <code>AudioContext</code> on page load, you’ll have to call
 <code>resume()</code> at some time after the user interacted with the page
-(e.g., user clicked a button).
+(e.g., user clicked a button). Alternatively, the <code>AudioContext</code> 
+will be resumed after a user gesture if <code>start()</code> is called on
+any attached node.
```

Comment:
Would it be worth mentioning the intent? ie. backward compatibliity

Reply:
I thought about that but opted to keep it simple, also didn't want to make it sound like developers didn't have to do anything (I didn't want developers to incorrectly jump to that conclusion if they were skimming)